helm/cluster-resources/values.yaml

```diff
@@ -14,3 +14,7 @@ image:
   pullPolicy: IfNotPresent
 
 cni: cilium
+
+cilium:
+  allowAllIngressFromHostIn:
+    - kube-system
```

Comment:
When would we add more namespaces here?

Reply:
Also it might be good to add a comment or change the naming to indicate that this is a list of namespaces.

Reply:
This is not really meant to be extended but I didn't want to name `allow-all-ingress-from-host-in-kube-system-cilium-np.yaml`

Let me know what you prefer.